cdk/domino_cdk/config/vpc.py

```diff
@@ -11,6 +11,10 @@ class VPC:
     id: vpc-abc123 - VPC id when using an existing VPC
     cidr: 10.0.0.0/16 - Primary CIDR range for VPC
                         NOTE: EKS needs _lots_ of IPs
+    private_cidr_mask: 19 - Size of private subnets. Should be large enough
```

Comment:
I'm confused. Is 19 a prefix, or a mask or a network size? It looks like a prefix but is called mask on left side and size of subnet on the right size. The network size is reverse logarithmic proportional to the prefix length. I.e. prefix length 32 means network size of 1.

Reply:
Subnet size in CIDR notation. I updated the phrasing to specify that it's CIDR size.